pkg/jobs/jobs.go

```diff
@@ -68,7 +68,9 @@ func (m *Manager) DeleteFor(vmiCrName types.NamespacedName) error {
 		return err
 	}
 	if job != nil {
-		return m.client.Delete(context.TODO(), job)
+		foreground := metav1.DeletePropagationForeground
+		opts := &client.DeleteOptions{PropagationPolicy: &foreground}
+		return m.client.Delete(context.TODO(), job, opts)
```

Comment:
Can it be verified as part of a functional test ? Else, an automation test should verify there are no leftovers after the

Reply:
Yes, I'm working on functional tests.